imagersite/imager_images/tests.py

```diff
@@ -128,3 +133,253 @@ def test_album_deletion_on_user_deletion(self):
         self.user1.delete()
         self.assertEqual(Album.objects.count(), 0)
 
+
+@override_settings(DEBUG=True)
+class LiveServerTest(StaticLiveServerTestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        super(LiveServerTest, cls).setUpClass()
+        cls.browser = Browser()
```

Comment:
be aware that session state for this browser will be shared between tests.  By setting this up on the `setUpClass` method, you are breaking test isolation with respect to browser state.